src/agent/coverage/src/block.rs

```diff
@@ -40,12 +40,10 @@ impl CommandBlockCov {
 
     pub fn increment(&mut self, path: &ModulePath, offset: u64) -> Result<()> {
         if let Some(module) = self.modules.get_mut(path) {
```

Comment:
Now, when we _merge_ command coverage, we now add unknown modules to the overall coverage. Here, we drop increment attempts for unknown modules. The reason this changed is that although right now we expect to precompute possible coverage locations, we'd like this format to support coverage discovery.

We should probably create a new (empty) `ModuleCov` for `path` as-needed.